### PR TITLE
Speed up Application Platform system test

### PR DESCRIPTION
**Pull request by gazayas**
Addresses the TODO in `test/system/application_platform_test.rb`

## Before
```
Finished in 37.252439s, 0.0268 runs/s, 0.2148 assertions/s.
1 runs, 8 assertions, 0 failures, 0 errors, 0 skips
```

## After
```
Finished in 21.607441s, 0.0463 runs/s, 0.3702 assertions/s.
1 runs, 8 assertions, 0 failures, 0 errors, 0 skips
```

### CODE REVIEW

File: test/system/application_platform_test.rb [hunk 1]
```diff
@@ -69,8 +69,7 @@ def within_current_memberships_table
         refute page.has_content?("Test Platform Application")
       end
       # The tombstones partial won't be rendered if there aren't any tombstoned memberships.
-      # TODO: This eats up a lot of time, look for a better way to check for this.
-      refute page.has_css?("tbody[data-model='Membership'][data-scope='tombstones']")
+      refute all("h2").map { |tag| tag.text }.include?("Former Team Members")
```
Comment (kaspth) [inline test/system/application_platform_test.rb:72]: I wonder if we could get this with just:

```suggestion
      assert_no_text "Former Team Members"
```

**Review by kaspth — APPROVED** [review]
Nice speed boost 🚀

File: test/system/application_platform_test.rb (hunk 1 above)
Comment (kaspth) [reply test/system/application_platform_test.rb:72]: Or maybe this?

```suggestion
      assert_no_selector "h2", text: "Former Team Members"
```

**Comment by gazayas** [comment]
> Nice speed boost 🚀

Thanks! I like the second suggestion so I went ahead and changed it in the last commit.

```
Finished in 21.167257s, 0.0472 runs/s, 0.3779 assertions/s.
1 runs, 8 assertions, 0 failures, 0 errors, 0 skips
```

**Pull request merged**